Tests/AK/TestCircularBuffer.cpp

```diff
@@ -348,3 +348,18 @@ TEST_CASE(offset_of_with_until_and_after_wrapping_around)
     result = circular_buffer.offset_of("Well "sv, 14, 19);
     EXPECT_EQ(result.value_or(42), 14ul);
 }
+
+BENCHMARK_CASE(looping_copy_from_seekback)
+{
+    auto circular_buffer = MUST(CircularBuffer::create_empty(16 * MiB));
+
+    {
+        auto written_bytes = circular_buffer.write("\0"sv.bytes());
+        EXPECT_EQ(written_bytes, 1ul);
+    }
+
+    {
+        auto copied_bytes = TRY_OR_FAIL(circular_buffer.copy_from_seekback(1, 15 * MiB));
+        EXPECT_EQ(copied_bytes, 15 * MiB);
+    }
+}
```

Comment:
i might be wrong but i feel you could test this case on bytes. 😉 (i.e. remove `* MiB`).

Reply:
This is a benchmark case, and we need large sizes for this to get some kind of stable performance numbers. Even when using `KiB` we were still in the 0-1 millisecond range for the unoptimized case, which doesn't give us any numbers to work with. Therefore, I decided to use `MiB` instead.

Reply:
(darn…! missed that! 😣)